sopel/modules/reddit.py

```diff
@@ -113,20 +113,58 @@ def rpost_info(bot, trigger, match):
         return NOLIMIT
 
 
-# If you change this, you'll have to change some other things...
-@commands('redditor')
-@example('.redditor poem_for_your_sprog')
-def redditor_info(bot, trigger, match=None):
+def subreddit_info(bot, trigger, match, commanded=False):
+    """Shows information about the given subreddit"""
+    r = praw.Reddit(
+        user_agent=USER_AGENT,
+        client_id='6EiphT6SSQq7FQ',
+        client_secret=None,
+    )
+    try:
+        r.subreddits.search_by_name(match, exact=True)
+        s = r.subreddit(match)
+        link = "https://www.reddit.com/r/" + s.display_name
+
+        tz = time.get_timezone(bot.db, bot.config, None, trigger.nick,
```

Comment:
Let's have all arguments on the same line:

```
tz = time.get_timezone(
    bot.db, bot.config, None, trigger.nick, trigger.sender)
```

Or everything aligned (but I don't like it):

```
tz = time.get_timezone(bot.db,
                       bot.config,
                       None,
                       trigger.nick,
                       trigger.sender)
```

Reply:
Actually, sinec this timezone stuff is now used twice, I was considering a seperate function for it.

Reply:
I'm on mobile atm, but I also believe that I just copy pasted that timezone code from the reddit post function.

I'd actually reccomend a seperate PR to clean this module completely, as it seems out of scope here. (I can do that later)

Ultimately, the module should be extracted into it's own package.

Reply:
I'm on mobile atm, but I also believe that I just copy pasted that timezone code from the reddit post function.

I'd actually reccomend a seperate PR to clean this module completely, as it seems out of scope here. (I can do that later)

Ultimately, the module should be extracted into it's own package.

Reply:
I know it was already there, but since you refactor the code a bit, I think it's the right time to improve its style too! Don't worry, you already did good a work here! ;)